src/sentry/db/models/mixins.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+__all__ = ('OrganizationBoundMixin',)
+
+from .fields import FlexibleForeignKey
+from .manager import OrganizationBoundManager
+
+
+class OrganizationBoundMixin(object):
+    organization = FlexibleForeignKey('sentry.Organization')
```

Comment:
likely will need to tweak this given some models use an integer and some use a foreignkey

might just create two diff mixins for it